src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -139,20 +147,23 @@ TargetWorkerSetNodeList(TargetWorkerSet targetWorkerSet, LOCKMODE lockMode)
 
 
 /*
- * SendBareCommandListToWorkers sends a list of commands to a set of target
+ * SendBareCommandListToMetadataWorkers sends a list of commands to metadata
  * workers in serial. Commands are committed immediately: new connections are
  * always used and no transaction block is used (hence "bare"). The connections
  * are made as the extension owner to ensure write access to the Citus metadata
  * tables. Primarly useful for INDEX commands using CONCURRENTLY.
  */
 void
-SendBareCommandListToWorkers(TargetWorkerSet targetWorkerSet, List *commandList)
+SendBareCommandListToMetadataWorkers(List *commandList)
```

Comment:
Minor: It would kind of make sense for metadata syncing functions to live in metadata_sync.c, but seems ok to keep them here for the moment to keep the change small.

Reply:
Should I create `SendBareCommandListInternal()` for consistency with the other APIs where `SendBareCommandListToMetadataWorkers ` would only call `SendBareCommandListInternal` with `WORKERS_WITH_METADATA `?

Reply:
Could be nice, don't feel strongly about it.